background/main.ts

```diff
@@ -845,27 +846,48 @@ export default class Main extends BaseService<never> {
   async connectIndexingService(): Promise<void> {
     this.indexingService.emitter.on(
       "accountsWithBalances",
-      async (accountsWithBalances) => {
+      async ({ balances, addressOnNetwork }) => {
         const assetsToTrack = await this.indexingService.getAssetsToTrack()
+        const trackedAccounts = await this.chainService.getAccountsToTrack()
+        const allTrackedAddresses = new Set(
+          trackedAccounts.map((account) => account.address)
```

Comment:
Are these addresses normalized in chainService? 
```suggestion
          trackedAccounts.map((account) => normalizeEVMAddress(account.address))
```

Reply:
Good question, all the wallet account addresses are normalized before reaching the chain db [here](https://github.com/tallycash/extension/blob/272e38f90c0e3dff95ad78f4de7c42ec5f89ada5/background/redux-slices/accounts.ts#L418) with the exception of ledger addresses which are normalized after derivation.